src/rules/no-non-numeric-dimensions/__tests__/index.js

```diff
@@ -19,6 +19,11 @@ testRule(rule, {
     {
       code: "$pad: 2; $doublePad: #{$pad}pxx;",
       description: "does not report lint when no understood units are used"
+    },
+    {
+      code: `$pad: "2";
+      $string: "#{$pad}px";`,
+      description: "does not report lint when no understood units are used"
```

Comment:
Maybe "when the string is quoted"?

Reply:
![image](https://user-images.githubusercontent.com/1325798/63060702-d393b780-bea7-11e9-9703-41b045481cbb.png)